src/style.css

```
@@ -0,0 +1,23 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 13px;
+}
+
+h1 {
+  font-size: 17px;
+  font-weight: bold;
+}
+
+h2 {
+  font-size: 15px;
+  font-weight: normal;
+  margin: 1em 0;
+}
+
+a {
+  color: green;
+}
+
+.title {
```

All paragraphs should be without outer spacings too
```suggestion
h1,
p {
```
You can remove the class `title`